.storybook/main.js

```diff
@@ -42,3 +71,9 @@ module.exports = {
     },
   },
 }
+
+// Сторибук использует абсолютные пути в моно репах
+// Функция автоматически сгенерена при миграции до 8 версии
```

Comment:
```suggestion
// Функция автоматически сгенерирована при миграции до 8 версии
```

Пиши слова правильно, иначе игнор cspell будет постоянно пополняться


И не совсем понятен смысл данного сообщения